magstock/__init__.py

```diff
@@ -57,7 +65,7 @@ def food_cost(self):
 
     @property
     def auto_food(self):
-        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE] or hasattr(self, "auto_food_extra") and self.auto_food_extra
```

Comment:
so this "auto_food_extra" is implemented by the band plugin, and you can see that here: https://github.com/magfest/bands/pull/43/files

basically the band plugin just says that any Attendee who is a member of a Group that is a Band gets free food.